SU-HSDO/suhumsci
# STN-856:  Accordion credit caption overflow bug

# READY FOR REVIEW

## Summary
The caption credit when added as a caption on an image within an accordion paragraph component was overflowing beyond the space of the image.

Example:
![image](https://user-images.githubusercontent.com/9355698/131055814-e858bbc6-729f-4b18-9dd3-22968fcb52f5.png)

JIRA Card(s): https://sparkbox.atlassian.net/browse/STN-856 (linked to 👉🏻 ) https://sparkbox.atlassian.net/browse/STN-847

## Need Review By (Date)
9/27

## Urgency
medium

## Steps to Test
1. npm run test
2. Create a Flexible Page, add an Accordion paragraph component.
3. Within that Accordion add an image to the text area
4. After inserting the image, Click the "Edit Media" button, check the "Caption" option and enter some text in for the image caption.
5. Save the page and check out the image, the credit/caption gradient should now display correctly within the image `<figure>` element.
6. Verify the caption/credit stays contained within the image when it's floated left or right as well, you can access this when you hit the Edit Media button on the Image. **(You will see the accordion container does collapse when floating the left or right in an accordion. I created an additional bug card for this bug, it needs to be fixed in the `hs_layout` module `patterns`)**
6. Verify the caption/credit is unaffected in other locations where they can be added and other images in standard Text Area's on pages.
7. Verify the caption/credits display correctly on both the Traditional and Colorful themes.

Like this:
![image](https://user-images.githubusercontent.com/9355698/131055785-075899a3-4bd0-4141-a37c-41b2119c6acd.png)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
